package.json

```diff
@@ -30,30 +31,32 @@
   "author": "Simon Tsvilik",
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "^7.21.0",
-    "@babel/core": "^7.21.4",
-    "@babel/preset-env": "^7.21.4",
-    "@babel/register": "^7.21.0",
-    "@wdio/cli": "^7.30.2",
-    "@wdio/local-runner": "^7.30.2",
-    "@wdio/mocha-framework": "^7.30.2",
-    "@wdio/selenium-standalone-service": "^7.30.2",
-    "@wdio/spec-reporter": "^7.30.2",
+    "@babel/cli": "^7.23.0",
+    "@babel/core": "^7.23.2",
+    "@babel/preset-env": "^7.23.2",
+    "@babel/register": "^7.22.15",
```

Comment:
Can we switch to TypeScript by chance? It would align with how we develop other services too.

Reply:
Definitely. It was something I was considering but wasn't sure if that was the intended direction. Will have it on my todo list

Reply:
It seems this will be a bit of a larger rewrite anyway so let's add it right away.

Reply:
Updated now to include typescript in the src files, and removed babel references